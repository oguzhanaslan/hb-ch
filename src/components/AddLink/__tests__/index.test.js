import React from 'react';
import ReactDOM from 'react-dom';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure, mount } from 'enzyme';
import renderer from 'react-test-renderer';

configure({ adapter: new Adapter() });

describe('Footer Test', () => {});
