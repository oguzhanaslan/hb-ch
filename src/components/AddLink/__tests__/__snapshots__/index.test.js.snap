// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test should render snapshot 1`] = `
<div>
  <form
    className="form"
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <input
        name="title"
        onChange={[Function]}
        placeholder="Title"
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <input
        name="type"
        onChange={[Function]}
        placeholder="Type"
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <button
        className="submit"
        type="submit"
      >
        Send
      </button>
    </div>
    <div
      className="form-group"
    >
      <button
        className="reset"
        onClick={[Function]}
        type="button"
      >
        Reset
      </button>
    </div>
  </form>
</div>
`;
